class/defaults.yml

```diff
@@ -44,6 +44,7 @@ parameters:
         appuio_cloud_memory: true
         appuio_cloud_loadbalancer: true
         appuio_cloud_persistent_storage: true
+        appcat_postgresql_vshn_standalone: true
```

Comment:
Not sure if we should actually enable this backfill job by default.

Reply:
Since VSHN-specific, non-APPUiO logic of AppCat has already been integrated into appuio/appuio-cloud-reporting, I don't see any issues with enabling the backfill job by default.